filetracker/interaction_test.py

```diff
@@ -13,7 +13,7 @@
 from wsgiref.simple_server import make_server
 
 from filetracker.client import Client, FiletrackerError
-from filetracker.servers.files import LocalFileServer
+from filetracker.servers.files import FileTrackerServer
```

Comment:
This might seem minor, but I'd prefer to use `Filetracker` (as one word) everywhere for consistency.